sample_project/metadata_fields.yaml

```diff
@@ -1,10 +1,10 @@
 File:
     Type: string
     LongName: name of the video file
-    Description: File naming system = commonnamesubjectletter_treatment_videonumber.avi (eg, jwaspB_rock_2.avi = The second video of the second jewel wasp subject, B, with a treatment of "rock", which is described further in the "Treatment_description" column). To view the file names within the master-list.xlsx, you will need version 2020 or later, because I employ sequence arrays - a command only available after 2020).
+    Description: File naming system = commonnamesubjectletter_treatment_videonumber.avi (eg, jwaspB_rock_2.avi = The second video of the second jewel wasp subject, B, with a treatment of "rock", which is described further in the "Treatment_description" column). To view the file names within the master-list.xlsx, you will need version 2020 or later, becuase I employ sequence arrays - a command only available after 2020).
```

Comment:
```suggestion
    Description: File naming system = commonnamesubjectletter_treatment_videonumber.avi (eg, jwaspB_rock_2.avi = The second video of the second jewel wasp subject, B, with a treatment of "rock", which is described further in the "Treatment_description" column). To view the file names within the master-list.xlsx, you will need version 2020 or later, because I employ sequence arrays - a command only available after 2020).
```